assets/templates/partials/filter.tmpl

```diff
@@ -137,6 +139,108 @@
     </noscript>
 </form>
 
+{{ if $population_types }}
+<form id="filterForm" method="get" class="js-auto-submit__form">
+    <input type="hidden" name="q" value="{{ $query }}"/>
+    <input type="hidden" name="sort" value="{{$sort}}"/>
+    <fieldset class="ons-fieldset">
+        <details id="collapsibleContentType" class="ons-collapsible ons-js-collapsible" data-btn-close="Hide this" data-open="true">
+            <summary class="ons-collapsible__heading ons-js-collapsible-heading">
+                <h2 class="ons-collapsible__title">
+                    <legend class="ons-fieldset__legend">
+                        {{ localise "PopulationTypes" $lang 1 }}
+                    </legend>
+                </h2>
+                <span class="ons-collapsible__icon">
+                    {{ template "icons/chevron-right" }}
+                </span>
+            </summary>
+            <div id="collapsible-content" class="ons-collapsible__content ons-js-collapsible-content">
+                <p class="ons-checkboxes__label">Show only:</p>
+        {{ range $index, $populationFilter := $population_types }}
+            <div class="ons-checkboxes__items">
+                <span class="ons-checkboxes__item ons-checkboxes__item--no-border">
+                    <span class="ons-checkbox ons-checkbox--no-border population-types">
+                        <input type="checkbox" id="population-group-{{ $index }}"
+                            class="ons-checkbox__input ons-js-checkbox"
+                            name="populations"
+                            value="{{ print $populationFilter.LocaliseKeyName }}" aria-controls="population-group-{{ $index }}-other-wrap"
+                            data-gtm-label="{{ $populationFilter.LocaliseKeyName }}"
+                            {{ if $populationFilter.IsChecked }}
+                                checked
+                            {{ end }}
+                            {{ if eq $populationFilter.Count 0 }} disabled aria-disabled="true" {{ end }}
+                        >
+                        <label class="ons-checkbox__label" for="population-group-{{ $index }}" id="population-group-{{ $index }}-label">
+                            {{ localise $populationFilter.LocaliseKeyName $lang 4 }} ({{ $populationFilter.Count }})
```

Comment:
Given we're unlikely to get welsh translations, is it easier to just render whatever the Search API passes?